### PR TITLE
Log disk stuff

### PR DESCRIPTION
**Pull request by matthchr**
Closes #[issue number]

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**How does this PR make you feel**:
gif

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains tests

### CODE REVIEW

File: .github/workflows/pr-validation.yml [hunk 1]
```diff
@@ -55,6 +55,9 @@ jobs:
           fetch-depth: 0 # required to access tags
           submodules: 'true'
 
+      - name: Force docker to SSD
```
Comment (super-harsh) [inline .github/workflows/pr-validation.yml:58]:
```suggestion
      - name: Force docker to use SSD
```

File: .github/workflows/pr-validation.yml [hunk 2]
```diff
@@ -121,6 +124,9 @@ jobs:
         
       - name: Build docker image & build configuration YAML
         run: |
+          df -H
+          docker info
+          docker image list
```
Comment (super-harsh) [inline .github/workflows/pr-validation.yml:129]: Do we still need these logs here to be merged?

**Review by super-harsh — APPROVED** [review]

File: .github/workflows/pr-validation.yml (hunk 2 above)
Comment (theunrepentantgeek) [reply .github/workflows/pr-validation.yml:129]: For now, it is probably good to keep these. Maybe we remove them later.

**Review by theunrepentantgeek — APPROVED** [review]

File: .github/workflows/pr-validation.yml (hunk 2 above)
Comment (super-harsh) [reply .github/workflows/pr-validation.yml:129]: In that case, I'd suggest leaving a TODO here so we know this is just test stuff and can be removed later.

Comment (matthchr) [reply .github/workflows/pr-validation.yml]: I've removed them now

**Pull request merged**